vcr/vcr_test.go

```diff
@@ -106,7 +106,7 @@ func TestVCR_Search(t *testing.T) {
 	if !assert.NoError(t, err) {
 		return
 	}
-	q.AddClause(concept.Eq("company.name", "Because we care BV"))
+	q.AddClause(concept.Eq("organization.name", "Because we care BV"))
```

Comment:
Shouldn't this be human.eyeColour?

Reply:
Test doesn't care, apparently...

I figured this is a bug (or a feature?): when there's no (matching?) clauses it appears all entries match?

Reply:
wrong setup of test I guess. This is also why your UI returns all matches without query. Since the type of credential is in the index. It then uses a partial index to find all.